### ISSUE TITLE
VOD chat stream ends before it should 

### ISSUE THREAD

**Issue by ancho85** [fgl27/smarttv-twitch#275]
**Describe the bug**
By watching a VOD and at the last 1 or 2 minutes the chat window informs 'The chat has ended', however by rewinding a couple of seconds the chat regains its correct stream as expected and ends when it really has to end.

**To Reproduce**
1. Go to any VOD with chat active until the very end of the stream
2. See 'The chat has ended' before it actually ends 
3. Go back a couple of seconds, the chat reloads with the chat stream fixed

**Expected behavior**
Chat should continue until the end

**Screenshots**

https://github.com/fgl27/smarttv-twitch/assets/3135981/fe340507-a066-416f-9a4a-5ad6106a07d6

In the video attached the chat shows this behavior, I'm going back 30 seconds and the chat resumes.

**Device:**

TV Model: UN50NU7095
Os version Tizen 1334

**Comment by fgl27** [comment]
Will revise this and let you know.

**Comment by fgl27** [comment]
I'm not able to reproduce this, is it still happening?

**Comment by ancho85** [comment]
https://github.com/fgl27/smarttv-twitch/assets/3135981/0af103d3-8a91-4af2-91b5-53cff769e78c
Yes, just tested again and it still happens. Maybe it won't happen if you fast forward to the end, but by watching it all it happens. This VOD shows the "chat ended" message 2 minutes before it actually ends.

**Comment by fgl27** [comment]
OK will try again, thanks.

**Comment by fgl27** [comment]
How many minutes from the end I can fast-forward to see the problem? I can't wat hours to see a problem

**Comment by ancho85** [comment]
I'm thinking 5 minutes before it ends, however as I said, maybe that fast-fowarding to the end will make it sync and won't show the problem. I'll have to test it again by doing that.

**Comment by fgl27** [comment]
I have found something to improve, I assume it fixes the issue.

as is a little hard to test I assume, but please let me know if you see the behavior happening again.

**Comment by ancho85** [comment]
Thank you! Will this require something to be updated on my end?

**Comment by fgl27** [comment]
No, you just need to close the app and re open for it to update, in case it was open during the time I posted and now, if not you already received the update

**Comment by ancho85** [comment]
Just tested and I can confirm the chat ended with the VOD as expected. Thank you Felipe
